controller.py

```diff
@@ -12,17 +12,24 @@
 import os
 
 from models.resnet import *
+from models.mvcnn import *
 import util
 from logger import Logger
 from custom_dataset import MultiViewDataSet
 
+MVCNN = 'mvcnn'
+RESNET = 'resnet'
+MODELS = [RESNET,MVCNN]
```

Comment:
Both models are "mvcnn" but based on resnet and alexnet. If you call one resnet I would also call mvcnn alexnet.

Reply:
Yes you are correct

Reply:
Anyway, thanks for your contributions, I merged it and did some minor fixes with the defaults :+1:

Reply:
Sure thanks for setting up the initial repo. It was very helpful.